packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -414,6 +422,7 @@ export default mixins(externalHooks, nodeHelpers).extend({
 			COMMUNITY_NODES_INSTALLATION_DOCS_URL,
 			CUSTOM_NODES_DOCS_URL,
 			MAIN_NODE_PANEL_WIDTH,
+			hiddenIssuesInputs: [] as string[],
```

Comment:
why do we need this at node settings level? cannot we track this for each parameter within `ParameterInputFull` instead?

Reply:
If I'd put it in `ParameterInputFull`, I would have to re-implement this logic also in `NodeCredentials` component since we pass it via slots to `ParameterInputList`. This way we only need to track the changes on the container component(`NodeSettings`)